src/layouts/BaseLayout/BaseLayout.jsx

```diff
@@ -11,8 +11,8 @@ import GlobalTopBanners from 'shared/GlobalTopBanners'
 import LoadingLogo from 'ui/LoadingLogo'
 
 import { useUserAccessGate } from './hooks/useUserAccessGate'
+import OnboardingOrChildren from './OnboardingOrChildren'
```

Comment:
Should probably lazily load this and re-wrap the component in a suspense boundary.